src/AndroidDebugLauncher/AndroidLaunchOptions.cs

```diff
@@ -38,6 +38,16 @@ public AndroidLaunchOptions(MICore.Xml.LaunchOptions.AndroidLaunchOptions xmlOpt
             else
                 this.IntermediateDirectory = GetOptionalDirectoryAttribute(xmlOptions.IntermediateDirectory, "IntermediateDirectory");
 
+            if (targetEngine == TargetEngine.Java)
+            {
+                this.JVMHost = LaunchOptions.RequireAttribute(xmlOptions.JVMHost, "JVMHost");
+                this.JVMPort = xmlOptions.JVMPort;
+                this.SourceRoots = LaunchOptions.RequireAttribute(xmlOptions.SourceRoots, "SourceRoots").Split(new char[] { ';' });
```

Comment:
SourceRoots can't be required because we currently don't use it for native debugging on android.